app/common/src/text/english.json

```diff
@@ -1231,6 +1231,9 @@
   "trialEnded": "Your cloud access trial period has endend.",
   "trialEndedExplanation": "You can proceed to checkout and extend your access or you can downgrade to Community plan.",
   "trialEndedWarning": "Moving to Community plan will discard your access to all of your cloud assets. We will permanently remove them after 90 days.",
+  "downgradedTitle": "Plan downgraded",
```

Comment:
I suggest changing the title to “Cloud Assets Removal,” which will better catch the reader's attention and make them less likely to dismiss the modal without reading.